app/controllers/core_induction_programmes/modules_controller.rb

```diff
@@ -79,4 +80,8 @@ def find_next_module
   def fill_data_layer
     data_layer.add_module_info(@course_module)
   end
+
+  def cip_is_ambition?
+    @course_module.course_year.core_induction_programme.name == "Ambition Institute"
+  end
```

Comment:
I do not want to open that can of worms.

Instead of making our code aware of 'oh, you are displaying ambition, display it differently', please add a field called `page_header` to all modules, make it nullable, and in place of the page title display that - unless it is empty, then display the title.